apps/api/src/domain/models/vocabulary-list/commands/add-term-to-vocabulary-list/index-term-added-to-vocabulary-list-on-term-view.event-handler.ts

```diff
@@ -0,0 +1,48 @@
+import { Inject } from '@nestjs/common';
+import { CoscradEventConsumer, ICoscradEventHandler } from '../../../../../domain/common';
+import { isNotFound } from '../../../../../lib/types/not-found';
+import { ITermQueryRepository, TERM_QUERY_REPOSITORY_TOKEN } from '../../../term/queries';
+import {
+    IVocabularyListQueryRepository,
+    VOCABULARY_LIST_QUERY_REPOSITORY_TOKEN,
+} from '../../queries';
+import { TermAddedToVocabularyList } from './term-added-to-vocabulary-list.event';
+
+/**
+ * This is different from the `TermAddedToVocabularyListHandler` in that here
```

Comment:
This is a really interesting point. We might be ok with assuming a shared database in the **view** layer, where we denormalize, but we need to think this through. It's good to have both options in play for a while so we can weigh the pros and cons.

Reply:
We've thought this through and we are ok sharing the database in the view layer. If we ever want to decouple, we can simply write a different handler. Also, terms and vocabulary list are already part of the same subdomain.

Reply:
Update the comment.